src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -151,9 +152,11 @@
         input-value    (reagent/atom "")
         input-focused? (reagent/atom false)]
     (fn []
-      (let [selected-tab          (or (rf/sub [:wallet/send-tab]) (:id (first tabs-data)))
-            valid-ens-or-address? (boolean (rf/sub [:wallet/valid-ens-or-address?]))
-            {:keys [color]}       (rf/sub [:wallet/current-viewing-account])]
+      (let [selected-tab             (or (rf/sub [:wallet/send-tab]) (:id (first tabs-data)))
+            valid-ens-or-address?    (boolean (rf/sub [:wallet/valid-ens-or-address?]))
+            local-suggestions        (rf/sub [:wallet/local-suggestions])
+            local-suggestion-address (:full-address (first local-suggestions))
+            {:keys [color]}          (rf/sub [:wallet/current-viewing-account])]
```

Comment:
I know it's not from this PR, but subscribing to `:wallet/current-viewing-account` just to get the color is not good, especially because that sub and its inputs signals are quite heavy in processing. The better approach would be to have a sub that returns only the color of the `current-viewing-account` from the input of sub `:wallet/accounts`.

Reply:
I'm sub-inspector Shivek. You have been found in possession of `{:keys [color]} (rf/sub ...)`,  which is an illegal substance. You have the right to resolve this. 

![image](https://github.com/status-im/status-mobile/assets/1925158/201c5040-11f2-4cd4-9385-b54e9a17fba6)

Reply:
Why isn't it called `wallet/current-viewing-account-color` ? 
The `->` is a convention for a pure transformation function.

Reply:
Done. Thanks for letting me know